jerry-core/mem/mem-context.c

```diff
@@ -0,0 +1,55 @@
+/* Copyright 2016 Samsung Electronics Co., Ltd.
+ * Copyright 2016 University of Szeged.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "mem-context.h"
+
+/** \addtogroup mem Memory allocation
+ * @{
+ *
+ * \addtogroup heap Context
```

Comment:
I have been thinking what is the best place for the context in Jerry, and for me the allocator is, since even if it is statically allocated, it is still "allocated". It could go to jrt, but for me that does not make sense. It cannot go to ecma, because mem cannot depend on ecma. I am open to better suggestions though.